docs/install/1-install-server.rst

```diff
@@ -357,8 +357,17 @@ to
   # IPv6 local connections:
   host    all             all             ::1/128                 md5
 
+Make sur JIT is disabled for the psql database as it will impact performances.
+To disable JIT, set
 
-Restart the PostgreSQL server to apply the changes made in the ``pg_hba.conf`` file:
+.. code-block:: text
+
+  # disable JIT
+  jit = off
+
+in ``/etc/postgresql/13/main/postgresql.conf``.
```

Comment:
I guess you are mentioning the path to the `postgresql.conf` file, like it is on Ubuntu. On RHEL, this file is present in a different location:
```suggestion
in ``/var/lib/pgsql/13/data/postgresql.conf``.
```